### PR TITLE
Query tables

### PR DESCRIPTION
**Pull request by J-C-S-V**
Hello dear reviewer 👋, I hope you're having a nice day today 😊

This is what I did on this branch:

- Use JOIN to create queries
- Answer the questions for analysis
- Create tables for owners and species
- Modify the animals table with 2 new fields `species_id ` and `owner_id `

These are the screenshots for the queries:

How many animals are there per species
How many animals are there per species

List all animals owned by Dean Winchester that haven't tried to escape.
List all animals owned by Dean Winchester that haven't tried to escape

List all Digimon owned by Jennifer Orwell.
List all Digimon owned by Jennifer Orwell

List all owners and their animals, remember to include those that don't own any animal.
List all owners and their animals, remember to include those that don't own any animal

List of all animals that are pokemon (their type is Pokemon).
List of all animals that are pokemon (their type is Pokemon)

What animals belong to Melody Pond
What animals belong to Melody Pond

Who owns the most animals
Who owns the most animals

Regards! ✨

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @J-C-S-V, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README fle is not professional and missing relevant headers like `Key features`, here is a [link](https://github.com/microverseinc/readme-template) to the recommended template. NOTE: Only the FAQ is the optional header

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql
Comment (Kamba56) [inline queries.sql:1]: - [x] I noticed you didn't add all queries to add data. Kindly add all data queries to `data.sql` file.
image

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @J-C-S-V, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job adding owners and species table👏👏👏
- [x] Great job using the join clause

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (J-C-S-V) [reply queries.sql:1]: Hi @Kamba56,

Can you kindly add the ones that I missed and be more specific?. I don't see any missing on my side.

Thank you!

Comment (J-C-S-V) [reply queries.sql:1]: Thank you @Kamba56,

I missed to push that change on data.sql. I've already pushed the missing data 💪

**Review by danmainah — APPROVED** [review]
Hi @J-C-S-V , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ All data added correctly using data.sql

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**